rustfmt.toml

```diff
@@ -1,2 +1,2 @@
 write_mode = "Overwrite"
-
+closure_block_indent_threshold = 1
```

Comment:
I also prefer block indent, but it does mean that we end up with block indents in a couple of places where rustfmt could choose not to wrap and indent at all. The issue you're referring to I believe was fixed by https://github.com/rust-lang-nursery/rustfmt/pull/1495, at least since then I haven't been seeing those format errors.